functions.php

```diff
@@ -134,6 +134,8 @@ function twentyseventeen_fonts_url() {
 	return esc_url_raw( $fonts_url );
 }
 
+
+
```

Comment:
I had given space to create the function there, when I decided to change place forgot to delete the spaces, thanks for the warning.